### ISSUE TITLE
Circular Saw (on) is the ultimate butchering tool

### ISSUE THREAD

**Issue by DeltaEpsilon7787** [CleverRaven/Cataclysm-DDA#27732]
**Screenshots**  
![butchering quality 60](https://vgy.me/HouDsN.png)

**Expected behavior**  
I found it _really_ hard to butcher my neighbor's dog precisely with a circular saw, so it's objectively wrong.

**Versions and configuration(please complete the following information):**
 - Game Version: 8440

**Comment by FuelType-Memes** [comment]
> **Expected behavior**
> I found it _really_ hard to butcher my neighbor's dog precisely with a circular saw, so it's objectively wrong.

Are you sure the problem was with the saw and not the police?

Circular saw should have bad butchering quality, but still count as bone saw for large creatures

**Comment by DeltaEpsilon7787** [comment]
>Are you sure the problem was with the saw and not the police?
Yes, the police were not a problem. It only costed me a few battery charges to deal with it.

>Circular saw should have bad butchering quality, but still count as bone saw for large creatures
60 butchering quality is way beyond any other tool in the game. The highest I've seen besides it was 25 butchering quality.

**Comment by Barhandar** [comment]
>60 butchering quality is way above any other tool in the game. The highest I've seen besides it was 25 butchering quality.

30, hunting knife. And this is intended - circular saw was set to be so way before current butchering system, when the quality only affected how long it took. It's a battery-using tool that you have to activate, hence having it be the fastest made sense (also, it's mentioned to have a small blade and can cut both wood and metal, hence, it's either got several swappable blades or uses a solid one - you can't cut metal with blades for wood or stone - wrong shape and diamond involved would mess up the metal, respectively).

If it actually affects the yields too now, then yeah, it's excessive.

**Comment by DeltaEpsilon7787** [comment]
>If it actually affects the yields too now, then yeah, it's excessive.
Which is why I made this issue in the first place.
